ui/imports/shared/popups/ImageCropWorkflow.qml

```diff
@@ -39,7 +39,7 @@ Item {
 
         title: root.imageFileDialogTitle
         folder: root.userSelectedImage ? imageCropper.source.substr(0, imageCropper.source.lastIndexOf("/")) : shortcuts.pictures
-        nameFilters: [qsTr("Supported image formats (%1)").arg("*.jpg *.jpeg *.jfif *.webp *.png *.heif")]
+        nameFilters: [qsTr("Supported image formats (%1)").arg("*.jpg *.jpeg *.jfif *.webp *.png")]
```

Comment:
Now I wonder what `webp` and `jfif` is doing here... :thinking:

Reply:
Elsewhere in the code, we're doing:
```qml
nameFilters: [
                qsTr("Image files (%1)").arg(Constants.acceptedDragNDropImageExtensions.map(img => "*" + img).join(" "))
            ]
```

Perhaps it would be nice to use it here too

Reply:
probably, yes, will fix